docs/src/tutorial/accept_loop.md

```diff
@@ -6,23 +6,20 @@ First of all, let's add required import boilerplate:
 
 ```rust,edition2018
 # extern crate async_std;
-use std::net::ToSocketAddrs; // 1
 use async_std::{
-    prelude::*, // 2
-    task, // 3
-    net::TcpListener, // 4
+    prelude::*, // 1
+    task, // 2
+    net::{TcpListener, ToSocketAddrs}, // 3
```

Comment:
Not sure why these docs got modified by this PR. Can you try rebasing on the master branch or do whatever is necessary to remove these changes in docs?

Reply:
In this PR, implemented an async ToSocketAddrs, so I replaced std::net::ToSocketAddrs to async_std::net::ToSocketAddrs. If you want to show "we're able to use block API", I'm going to revert this.

Reply:
Oh I'm sorry, I wasn't taking a good look! Please ignore this comment :)